src/coreclr/debug/daccess/enummem.cpp

```diff
@@ -1558,6 +1562,28 @@ HRESULT ClrDataAccess::EnumMemCLRMainModuleInfo()
         status = E_UNEXPECTED;
     }
 
+#if defined(FEATURE_SINGLE_FILE_DIAGNOSTICS)
```

Comment:
Where/when is FEATURE_SINGLE_FILE_DIAGNOSTICS defined?

Reply:
Uncommited file based on the feature flag in src/coreclr/clrfeatures.cmake; I removed it as it doesn't make much sense to compile without.